Using-Mastodon/FAQ.md

```diff
@@ -64,75 +71,147 @@ Try [TootyFruity](https://play.google.com/store/apps/details?id=ch.kevinegli.too
+
+#### How do I report offensive content?
+Beneath each post, you will see three dots. Clicking on those will allow you to expand a post or report the content. When reporting content, select all posts that need moderator attention.
+
+#### How do I handle harassment?
+Harassment is taken seriously by the admins and moderators of instances and should be reported through the post-reporting system which can be found via the three dots under every post or writing your admin/mod directly.
+
+#### Do my Private Messages get to people on other instances?
+They should, yes. However, because of different privacy settings and other factors, your post might not remain private under certain circumstances.
+
+#### What does “Adjust Status Privacy” mean and how does it work?
+The **globe** icon under the toot area adjusts your status privacy by changing who can see your posts. This is what happens:
+
+| Privacy setting | Destination | Notes |
+| --------------- | ----------- | ----- |
+| Public          | global, public timeline | Federated. People on other instances will probably see it. |
+| Unlisted        | local public timeline | Can be boosted and thus seen on other instances.  |
+| Private         | only to your followers |  Cannot be boosted. |
+| Direct          | mentioned people only  | Similar to email |
+
+#### Can I use hashtags? Should I?
+Yes! Hashtags are tracked and are often fun, but some tags help people avoid triggering posts. This is especially appreciated on public posts of #POLITICS, #HEALTH, #DEPRESSION, or #LEWD OR #NSFW topics. Such posts are also what the Content Warning system was designed for.
+
 #### How do I get verified with a “✅”?
 “✅” is an emoji, *only* for lulz. There is no verification on Mastodon. You can copy and paste “✅” into your bio if you wish, you cheeky monkey.
 
 #### Can I edit a toot?
-No, sorry. But you could delete and rewrite...
+No, sorry. But you could delete your toot and rewrite…
+
+#### If I delete a post, does it get deleted everywhere?
+Deleting a post will delete it from the local timeline. However, if it has already been boosted, it may still be seen and may federate.
 
 #### How do I view my favorite toots?
-Click "Getting Started," then "Favourites."
+Click **Getting Started**, then **Favourites**.
 
-![img](http://octodon.social/media/KKVbLbEdqa22oNNKQYg "Getting Started") ![img](https://octodon.social/media/wYG2bD7bAAplgZKkwH8 "Favourites")
+<img src="screenshots/toolbar-getting_started.png" alt="Getting Started Button" height="200"/>
+<img src="screenshots/getting_started-favourites.png" alt="Favourites" height="200"/>
 
 #### Can I quote a toot?
 Not currently. You must manually copy and paste the text to quote it.
 
+#### What if my instance shuts down?
```

Comment:
Probably not worth having a non-answer here. I'd imagine something like this might be better:

> Picking a Mastodon instance is about trust. Just like a Gmail user trusts that Google won't shut everything down tomorrow, you have to trust your instance administrator. If you don't trust anyone, you can always host your own instance!